src/total-value-and-liquidity/total-value-and-liquidity.js

```diff
@@ -0,0 +1,130 @@
+const ethers = require('ethers');
+const BigNumber = require('bignumber.js');
+const { getJsonRpcUrl } = require('forta-agent');
+const RollingMathLib = require('rolling-math');
+
+// get createAlert and dataFields
+const { createAlert, dataFields } = require('./common');
+
+// load required shared types
+const contractAddresses = require('../../contract-addresses.json');
+
+const { LendingPool: LendingPoolAddr, ProtocolDataProvider: DataProviderAddr } = contractAddresses;
```

Comment:
Agree with @arbitrary-kdawg comments about lowerCamelCase for the imported variable names.